### ISSUE TITLE
Doesn't work with strapi 4.13.6

### ISSUE THREAD

**Issue by damian-balas** [VirtusLab-Open-Source/strapi-plugin-navigation#377]
Hi! :)
After installing this plugin in strapi 4.13.6 the admin panel crashes (white screen) with this error:
![image](https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/39343843/05677398-73b4-4baf-ac7a-6b3dc34f5841)
![image](https://github.com/VirtusLab-Open-Source/strapi-plugin-navigation/assets/39343843/4d10f4da-14c2-4827-ac82-19b3a086194c)

This are my dependencies:
```json
"dependencies": {
    "@strapi/plugin-i18n": "4.13.6",
    "@strapi/plugin-seo": "^1.9.5",
    "@strapi/plugin-users-permissions": "4.13.6",
    "@strapi/strapi": "4.13.6",
    "pg": "^8.11.3",
    "strapi-plugin-navigation": "^2.2.13",
    "strapi-plugin-permalinks": "^2.0.1",
    "strapi-plugin-placeholder": "^4.4.0"
  },
```

Any idea what can cause this error? :)

I think it's somehow related to languages selected in `src/admin/app.tsx`

# UPDATE 
To fix this:
1. I had to restore `app.tsx`. 
2. Delete everything configured by default in `UI Navigation`. 
3. enable i18n in navigation configuration
4. uncomment languages I want in `app.tsx`
5. create new navigation

**Comment by cyp3rius** [comment]
Checking you case @damian-balas

**Comment by cyp3rius** [comment]
Fixed in version `2.2.14`
